### PR TITLE
update vllm to 0.3.2

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Upgrade vLLM to 0.3.2

### CODE REVIEW

**Review by maaquib — APPROVED** [review]

File: .github/workflows/lmi-dist-deps-build.yml [hunk 1]
```diff
@@ -87,10 +87,10 @@ jobs:
           cd llm-awq/awq/kernels && git checkout 8baf5dd9c3bfe8bdc5987f52ae4dffde7471346f
           pip wheel . --no-deps
           cp awq*.whl ../../../build_artifacts
-      - name: Build vllm 0.3.1 speculative decoding
+      - name: Build vllm 0.3.2 speculative decoding
         run: |
           . ./venv/bin/activate
-          git clone https://github.com/ymwangg/vllm -b specdec_v0.3.1
+          git clone https://github.com/ymwangg/vllm -b specdec_v0.3.2
```
Comment (rohithkrn) [inline .github/workflows/lmi-dist-deps-build.yml:93]: @chen3933 found a minor bug in this branch.

Comment (ymwangg) [reply .github/workflows/lmi-dist-deps-build.yml:93]: What's the issue?

Comment (chen3933) [reply .github/workflows/lmi-dist-deps-build.yml:93]: https://github.com/vllm-project/vllm/pull/2607/files

typo : worker.py - ln 209

Comment (ymwangg) [reply .github/workflows/lmi-dist-deps-build.yml:93]: Thanks! That's a good catch.

**Pull request merged**